aries-site/src/components/home/Quote.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import { Avatar, Box, Image, Paragraph, Stack, Text } from 'grommet';
+import { useDarkMode } from '../../utils';
+
+export const Quote = () => {
+  const darkMode = useDarkMode();
+
+  const quoteLines = `/static/images/quote-lines${
+    darkMode.value ? '-dark' : ''
+  }.svg`;
+
+  return (
+    <Box direction="row" gap="large" justify="center" pad="large">
+      <Box
+        align="center"
+        justify="center"
+        gap="medium"
+        background={{
+          image: `url(${quoteLines})`,
+          size: 'auto',
+          position: 'top left',
+        }}
+        pad={{ left: 'xlarge' }}
+        flex={false}
+      >
+        <Avatar src="/static/images/t_harms_bio_image.png" size="149px" />
+        <Box align="center">
+          <Text size="xlarge" weight="bold" color="text-strong">
+            Tim Harms
+          </Text>
+          <Text size="xlarge">HPE Head of Brand</Text>
+        </Box>
+      </Box>
+      <Stack>
+        <Box
+          border={{ size: '3px', color: 'text-weak' }}
+          pad="medium"
+          margin="small"
+          width={{ max: '678px' }}
+        >
+          <Paragraph size="xxlarge" textAlign="center">
+            The HPE Design System empowers developers, designers and others to
+            create consistent, accessible and flexible interfaces
```

Comment:
```suggestion
            create consistent, accessible and flexible interfaces.
```

Reply:
The Figma didn't have a period here but I agree this text should have it (unlike the highlight text which is often not a sentence.)